### DISCUSSION TITLE
LDAP support on Radicale 3.0

### DISCUSSION THREAD

**Discussion by EduardoATpt** [Kozea/Radicale#1377] [Ideas]
Hi there,

I have been using Radicale for quite a while and I was going to update the version that I have, but it seems that the LDAP support was removed... 
Can you confirm it? 

Is there anyway to use Radicale 3.0 with LDAP?

Thank you!
Eduardo

**Comment by isr4z6er** [comment]
Use this: https://github.com/cloudron-io/radicale-auth-ldap

**Comment by pbiering** [comment]
Improved since 3.3.0
